README.md

```diff
@@ -1 +1,9 @@
 # nc-time-axis
+
+Support for netcdftime axes in matplotlib.
```

Comment:
I do want the wording of this (and the docstring from `__init__.py`) to eventually change.